src/Tags.ts

```diff
@@ -0,0 +1,8 @@
+import { TagsClient } from './tags/Tags.js';
+import { Headers } from './types.js';
+
+export const Tags = (headers: Headers) => {
```

Comment:
shouldn't this go as part of the four big groups (engine, account, hub, backoffice?)

Reply:
I made a new group because tags is a separate app in api. Maybe put it under plugins?